static/src/stylesheets/module/facia-garnett/_container--video.scss

```diff
@@ -305,15 +291,53 @@ $video-width-desktop: 700px;
     }
 
     .fc-item__video-fallback,
-    .video-overlay,
-    .gu-media {
+    .gu-media,
+    .video-overlay {
         opacity: 1;
     }
 }
 
+.fc-container--video .youtube-media-atom__play-button.vjs-control-text,
+.media-primary .youtube-media-atom__play-button.vjs-control-text {
+    @include video-icon-size($vjs-medium-button-size, $vjs-medium-button-border-width);
+    @include mq(mobileLandscape) {
+        @include video-icon-size($vjs-large-button-size, $vjs-large-button-border-width);
+    }
+    @include mq(tablet) {
+        @include video-icon-size($vjs-x-large-button-size, $vjs-x-large-button-border-width);
+    }
+}
+
+.fc-item--list-media-mobile .youtube-media-atom__play-button.vjs-control-text {
+    @include mq($until: tablet) {
+        @include video-icon-size($vjs-small-button-size, $vjs-small-button-border-width);
+    }
+}
+
+.fc-item--standard-tablet .youtube-media-atom__play-button.vjs-control-text {
+    @include mq(tablet) {
+        @include video-icon-size($vjs-medium-button-size, $vjs-medium-button-border-width);
+    }
+}
+
+.fc-item--half-tablet .youtube-media-atom__play-button.vjs-control-text,
+.fc-item--full-media-75-tablet .youtube-media-atom__play-button.vjs-control-text {
+    @include mq($until: tablet) {
+        @include video-icon-size($vjs-large-button-size, $vjs-large-button-border-width);
+    }
+}
+
+.fc-item--three-quarters-tablet .youtube-media-atom__play-button.vjs-control-text {
+    @include mq(mobile) {
+        @include video-icon-size($vjs-large-button-size, $vjs-large-button-border-width);
+    }
+    @include mq(tablet) {
+        @include video-icon-size($vjs-x-large-button-size, $vjs-x-large-button-border-width);
+    }
+}
+
```

Comment:
Minor code style point, would it be possible to refactor this using the trailing `&` [parent selector](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#parent-selector)? I think this would make the code more readable.

```scss
.youtube-media-atom__play-button.vjs-control-text {
  .fc-container--video &,
  .media-primary & {
    @include video-icon-size($vjs-medium-button-size, $vjs-medium-button-border-width);
    // etc
  }

  .fc-item--list-media-mobile & {
    @include mq($until: tablet) {
      @include video-icon-size($vjs-small-button-size, $vjs-small-button-border-width);
    }
  }
  // etc
}
```

Let me know what you think, if you need help